index.js

```diff
@@ -12,10 +13,25 @@ import dbg from 'debug'
 const debug = dbg('jsolines')
 
 /**
- * Create a JSON isoline. Surface is a (possibly typed) array, width and height are its width and height, and cutoff is the cutoff.
- * It is possible to disable linear interpolation for debug purposes by passing interpolation: false
+ * Create a JSON isoline. Surface is a (possibly typed) array, width and height
+ * are its width and height, and cutoff is the cutoff. It is possible to disable
+ * linear interpolation for debug purposes by passing interpolation: false
  */
-export default function jsolines ({surface, width, height, cutoff, project, interpolation = true}) {
+export default function jsolines ({
+  surface,
+  width,
+  height,
+  cutoff,
+  project,
+  interpolation = true
+}: {
+  surface: Uint8Array,
+  width: number,
+  height: number,
+  cutoff: number,
+  project: Function,
```

Comment:
I think it may also have the benefit of affecting upstream apps that send typed functions.